src/app/form-page/form-page-styling/form-page-styling.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Store } from '@ngrx/store';
+import { takeUntil } from 'rxjs/operators';
+import { Observable, Subject } from 'rxjs';
+import { FormControl, FormGroup } from '@angular/forms';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+
+import * as fromFields from 'app/core';
+import { getCurrentFields, getGeneralStyle, getStaticFields } from 'app/core';
+import { FormElementModel } from 'app/shared/models/form-element.model';
+import { AddGeneralStyle } from 'app/core/fields/fields.action';
+
+@Component({
+  selector: 'app-form-page-styling',
+  templateUrl: './form-page-styling.component.html',
+  styleUrls: ['./form-page-styling.component.scss']
+})
+
+export class FormPageStylingComponent implements OnInit, OnDestroy {
+
+  public fields$: Observable<any>;
+  public ngUnsubscribe$ = new Subject<void>();
+
+  public form = new FormGroup({});
+  public defaultFields: FormElementModel[];
+  public panelOpenStyleList = false;
+  public panelOpenGeneralStyle = false;
+  public generalStyle$: Observable<any>;
+
+  constructor(
+    private storeFields: Store<fromFields.State>
+  ) {}
+
+  ngOnInit(): void {
+    this.fields$ = this.storeFields.select(getCurrentFields).pipe(takeUntil(this.ngUnsubscribe$));
```

Comment:
no need to use takeUntil(this.ngUnsubscribe$) if there is no subscriptions
```suggestion
    this.fields$ = this.storeFields.select(getCurrentFields));
```